ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
 
-See the [ADA support page on how Databricks works](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works).
+Migration to Databricks offers a lot of potential benefits to analysts. These include:
 
-## Differences between IDEs and Notebooks
+* Having all data together in one place and being able to access it via one interface, rather than split across separate areas requiring separate access permissions and pieces of software 
 
-RStudio is an IDE (Integrated Development Environment) whereas Databricks makes use of Notebooks, where you write your code in blocks contained in individual cells. There are advantages and disadvantages to both methods, and [this NHS-R article](https://nhsdigital.github.io/rap-community-of-practice/implementing_RAP/notebooks_versus_ide_development/#what-is-this-guide-for) outlines some of the differences you might come across when switching from using IDEs to Notebooks. 
+* If your scripts usually take a long time to run, cloud computing can speed up processing and reduce code running time. This is ideal for big data, scripts that require a lot of data transformation (e.g. complicated or large joins), and machine learning projects. 
+
+* If you regularly have to run the same code, access to workflows and code scheduling mean that you can set code to run at certain days or times to improve efficiency. Scheduled workflows will run even if your laptop is switched off.
+
+* Easier collaboration within workspaces
+
+* Better transparency of work being undertaken in the department by making use of the ADA shared reports area
 
 ---
 
-## Walkthrough a simple analysis
+### Making the most of Databricks 
+
+In order to take advantage of the benefits listed above, we recommend that you and your team:
+
+* Arrange data migration and access to Databricks. Onboarding is currently voluntary, so it's a great opportunity to move early and have more time to get familiar with the system and undertake training. Get in touch with [the ADA team](mailto:ADAPTTeam@education.gov.uk) to discuss onboarding with them. 
 
-See the [ADA support page on using R, python and SQL in Databricks](https://analytical-data-access.azurewebsites.net/get-support/using-r-python-and-sql-in-databricks)
+* Take part in Databricks training for R and/or SQL. R training is currently offered by the ADA team on a monthly basis and SQL training is in the process of being developed. Upcoming training is advertised on the [ADA user group Teams channel](https://teams.microsoft.com/l/team/19%3AGQW24K3V6M873QzUwpBIuyf8KZCxYByRawVsKQvpe-81%40thread.tacv2/conversations?groupId=c70bb59d-c5df-4af5-849f-36196ed14ba3&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9).
 
+* Be proactive in determining whether any changes will be necessary for your existing code and set time aside to make these changes - see the guidance in the [what this means for existing code](/ADA/ada.html#what-this-means-for-existing-code) section below. 
+
+::: {.callout-note}
```

Comment:
Interesting - this is a different style to what we've done elsewhere (Open data standards page has a tonne of them on) - I kind of like what you have here better. Should we add something to the style guide around these boxes and a consistent way to do them?

Reply:
Yes I think so - quarto reference document for them is here: https://quarto.org/docs/authoring/callouts.html